lib/babe/epoch_integration_test.go

```diff
@@ -180,7 +180,7 @@ func TestService_getLatestEpochData_epochData(t *testing.T) {
 	require.NoError(t, err)
 
 	data := &types.EpochData{
-		Authorities: auths[:3],
```

Comment:
why were we doing this before?

Reply:
Because in the `gssmr` spec file there are 9 authorities and this was to use only 3 out of the 9 in the spec file